### PR TITLE
Include demo for speeding up YOLOX with nebullvm inference accelerator

### PR DESCRIPTION
**Pull request by emilecourthoud**
This PR includes nebullvm among the tools for YOLOX deployment and optimization.

[nebullvm](https://github.com/nebuly-ai/nebullvm) is an open-source library designed to accelerate AI inference of deep learning models in a few lines of code. Nebullvm leverages state-of-the-art model optimization techniques such as deep learning compilers (TensorRT, Openvino, ONNX Runtime, TVM, TF Lite, DeepSparse, etc.), various quantization and compression strategies to achieve the maximum physically possible acceleration on the user's hardware.

During tests on various CPUs and GPUs and on YOLOX models of different sizes, nebullvm averaged a 2.2-fold acceleration in inference time.

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -14,4 +14,5 @@ tabulate
 pycocotools>=2.0.2
 onnx==1.8.1
 onnxruntime==1.8.0
+nebullvm>=0.4.4
```
Comment (FateScript) [inline requirements.txt:17]: Plz move this to your README of nebullvm, thanks.

**Review by FateScript — CHANGES_REQUESTED** [review]
PTAL @emilecourthoud

**Comment by emilecourthoud** [comment]
Hi @FateScript, I updated the PR according to your suggestions.

**Review by FateScript — APPROVED** [review]

**Comment by FateScript** [comment]
Well done @emilecourthoud :) Thanks for your contribution. LGTM.

**Comment by emilecourthoud** [comment]
Hello @FateScript, I noticed that the check fails. I have not changed any requirements, could you please check if this is a problem with YOLOX test code?

**Comment by FateScript** [comment]
@emilecourthoud Plz don't worry, I have fixed the workflow. I will rerun it.

**Pull request merged**